src/NLog/LogLevel.cs

```diff
@@ -251,7 +251,7 @@ public static LogLevel FromOrdinal(int ordinal)
                     return Off;
 
                 default:
-                    throw new ArgumentException("Invalid ordinal.");
+                    throw new ArgumentException($"Unknown loglevel: {ordinal.ToString()}.", nameof(ordinal));
```

Comment:
```suggestion
                    throw new ArgumentException($"Unknown log level ordinal: {ordinal.ToString()}.", nameof(ordinal));
```

Reply:
Well the Sonar Rule checks if the ArgumentException message contains the input-parameter in the text. So your suggestions wants this instead: `throw new ArgumentException($"Unknown log level {nameof(ordinal}: {ordinal.ToString()}.", nameof(ordinal));`

But I'm fine with just using ArgumentException-parameterName